nodeLib/buildNodePackage.nix

```diff
@@ -546,6 +544,7 @@ let
           echo "Symlinking current directory into node modules folder..."
           mkdir -p $(dirname $NODE_MODULES/$fullName)
           ln -s $(pwd) $NODE_MODULES/$fullName
+          runHook preShellPackageValidityCheck
```

Comment:
Not sure why this got added here, but it won't cause any harm even if not used. I assume it relates to the other PR.